File: src/types/marketplace-install-metadata.ts
export type MarketplaceInstallMetadata = {
  skillId: string | null;
  slug: string;
  name: string;
  githubUrl: string | null;
  skillUrl: string | null;
  remoteUpdatedAt: string | null;
  installTarget: string | null;
  installCollection: string | null;
  installedAt: string;
  installedPath: string;
  installer: string;
};

File: crates/skills-core/src/models/marketplace_install_metadata.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Deserialize, Serialize)]
#[serde(rename_all = "camelCase")]
pub struct MarketplaceInstallMetadata {
    pub skill_id: Option<String>,
    pub slug: String,
    pub name: String,
    pub github_url: Option<String>,
    pub skill_url: Option<String>,
    pub remote_updated_at: Option<String>,
    pub install_target: Option<String>,
    pub install_collection: Option<String>,
    pub installed_at: String,
    pub installed_path: String,
    pub installer: String,
}
